includes/admin/settings/class-wcmp-settings-data.php

```diff
@@ -176,10 +176,10 @@ static function() use ($section) {
                     if (isset($setting['condition']) && false === $setting['condition']) {
                         continue;
                     }
-                    $namePrefix           = $prefix ? "{$name}_" : '';
+                    $namePrefix           = false ? "{$name}_" : '';
```

Comment:
De $prefix is weg, hoewel ik dat eng vind vertrouw ik erop dat dat kan en blijkbaar niet werd gebruikt. Maar ik zou geen ternary laten staan met een boolean als conditie.